srcpkgs/pulseeffects-legacy/template

```diff
@@ -4,18 +4,16 @@ version=4.8.5
 revision=1
 wrksrc="pulseeffects-${version}"
 build_style=meson
-hostmakedepends="itstool pkg-config gettext"
+hostmakedepends="itstool pkgconf gettext"
```

Comment:
why?

Reply:
@paper42 because `pkgconf` is faster and better in any way: http://pkgconf.org/features.html

Reply:
almost every template uses pkg-config, so I think you should stick to the convention

Reply:
Who would win? "Rationality & utility" vs. "convention & tradition". I'll let Void members decide :)